### DISCUSSION TITLE
Home Assistant OS 11.5

### DISCUSSION THREAD

**Discussion by agners** [home-assistant/operating-system#3143] [Announcements]
## Changes

* Prepare 11.5.rc1 (#3110) @sairon
* Fix ARCH/SRCARCH for aarch64 in linux-check-dotconfig (#3098) @sairon
* Bump actions/cache from 3 to 4 (#3095) @dependabot
* Fix build_haos_builder step to fetch the correct version (#3091) @tompipe
* Dockerfile: migrate to trusted.gpg.d for docker repo keys (#3090) @darkxst
* Fix github build workflow when using self_signed_certs (#3071) @tompipe

## Home Assistant Operating System

* Use renamed MT7921/MT7922 options from linux-firmware upstream merge (#3106) @sairon
* Bump U-Boot to 2024.01 (#3096) @agners
* Bump buildroot to update linux-firmware (#3084) @sairon
* Merge Buildroot 2023.2.8 (#3060) @sairon

## Build System

* Fix path to VHDX artifact in OS build job (#3105) @sairon

## Raspberry Pi

* Add optional rpi-rf-mod dtoverlay block for RPi5 (#3080) @jens-maus
* Create empty overlays/README to fix DTB overlay loading on RPi 5 (#3107) @sairon
* Resolve warnings caused by incorrect kernel config settings (#3104) @sairon
* Bump U-Boot to 2024.01 (#3096) @agners
* Fix path to cmdline after successful RPi 5 tryboot (#3085) @sairon

## Home Assistant Yellow

* Resolve warnings caused by incorrect kernel config settings (#3104) @sairon

## Home Assistant Green

* Resolve warnings caused by incorrect kernel config settings (#3104) @sairon
* Bump U-Boot to 2024.01 (#3096) @agners
* Linux: Update kernel 6.1.74 (#3093) @sairon

## Open Virtual Appliance

* Add kernel options missing as a result of enabling CONFIG_EXPERT (#3114) @sairon
* Resolve warnings caused by incorrect kernel config settings (#3104) @sairon
* Linux: Update kernel 6.1.74 (#3093) @sairon

## Generic x86-64

* Add kernel options missing as a result of enabling CONFIG_EXPERT (#3114) @sairon
* Resolve warnings caused by incorrect kernel config settings (#3104) @sairon
* Linux: Update kernel 6.1.74 (#3093) @sairon

## Hardkernel ODROID

* Update ATF and DDR blob for Odroid M1, change 1056->1560 MHz RAM timing (#3125) @sairon
* Linux: Update kernel 6.1.75 (#3123) @sairon
* Disable Ethernet in U-Boot if ODROID-M1 (#3122) @agners
* Add kernel options missing as a result of enabling CONFIG_EXPERT (#3114) @sairon
* Resolve warnings caused by incorrect kernel config settings (#3104) @sairon
* Bump U-Boot to 2024.01 (#3096) @agners
* Linux: Update kernel 6.1.74 (#3093) @sairon

## ASUS Tinker

* Disable MMC environment on ASUS Tinkerboard (#3124) @agners
* Add kernel options missing as a result of enabling CONFIG_EXPERT (#3114) @sairon
* Resolve warnings caused by incorrect kernel config settings (#3104) @sairon
* Bump U-Boot to 2024.01 (#3096) @agners
* Linux: Update kernel 6.1.74 (#3093) @sairon

## Khadas VIM Series

* Add kernel options missing as a result of enabling CONFIG_EXPERT (#3114) @sairon
* Resolve warnings caused by incorrect kernel config settings (#3104) @sairon
* Bump U-Boot to 2024.01 (#3096) @agners
* Enable PCIe for Khadas VIM3 (#3092) (#3100) @agners
* Linux: Update kernel 6.1.74 (#3093) @sairon

## Generic aarch64

* Resolve warnings caused by incorrect kernel config settings (#3104) @sairon
* add missing ARMv8 32bit kernel support configs (#3099) @jens-maus
* Linux: Update kernel 6.1.74 (#3093) @sairon

<hr /><em>This discussion was created from the release <a href='https://github.com/home-assistant/operating-system/releases/tag/11.5'>Home Assistant OS 11.5</a>.</em>

**Comment by fonsuak** [comment]
Has anyone tested this version on a Raspberry PI5? Boot loop bug seems fixed but if someone has tried the update already please let me know how it went. I am still running 11.3 rc3. Thank you.
